### PR TITLE
Make value hashing consistent with `pg_diffix`.

### PR DESCRIPTION
**Pull request by cristianberneanu**

### CODE REVIEW

File: src/OpenDiffix.Core/Aggregator.fs [hunk 1]
```diff
@@ -17,7 +17,8 @@ let cloneHashSet (hashSet: HashSet<'T>) = HashSet<'T>(hashSet, hashSet.Comparer)
 let private invalidArgs (values: Value list) =
   failwith $"Invalid arguments for aggregator: {values}"
 
-let private hashAid (aidValue: Value) = Value.hash aidValue
+let private hashAid (aidValue: Value) =
+  aidValue |> Value.toString |> Hash.string
```
Comment (pdobacz) [inline src/OpenDiffix.Core/Aggregator.fs:21]: how consistent is consistent we desire here? Off the top of my head, firstly, I think `reference` appends `L` to the integers and floating point formatting rules might be different.

I'm guessing we have no use-case yet for exact match of the hashes between the 2 implementations, so we don't care about those details, but just making sure.

Comment (pdobacz) [reply src/OpenDiffix.Core/Aggregator.fs:21]: Actually, I'm confused, it appears that we're hashing memory bytes in `pg_diffix`, not string representations, or am I missing something?

**Review by pdobacz — DISMISSED** [review]
Codewise LGTM, but maybe I am missing something, left one question to clarify.

File: src/OpenDiffix.Core/Aggregator.fs (hunk 1 above)
Comment (cristianberneanu) [reply src/OpenDiffix.Core/Aggregator.fs:21]: Good catch, we hash integers directly in `pg_diffix`.

**Pull request merged**